index.php

```diff
@@ -344,41 +344,42 @@
 	//		}
 	//	}
 
-		if(($_SERVER['REQUEST_METHOD'] === 'POST') && (! $a->error)
-			&& (function_exists($a->module . '_post'))
+		$func = $a->module . '_post';
+		if (($_SERVER['REQUEST_METHOD'] === 'POST') && (! $a->error)
+			&& (function_exists($func))
 			&& (! x($_POST,'auth-params'))) {
 			call_hooks($a->module . '_mod_post', $_POST);
-			$func = $a->module . '_post';
 			$func($a);
 		}
 
-		if((! $a->error) && (function_exists($a->module . '_afterpost'))) {
-			call_hooks($a->module . '_mod_afterpost',$placeholder);
-			$func = $a->module . '_afterpost';
+		$func = $a->module . '_afterpost';
+		if ((! $a->error) && (function_exists($func))) {
+			call_hooks($a->module . '_mod_afterpost', $placeholder);
 			$func($a);
 		}
 
-		if((! $a->error) && (function_exists($a->module . '_content'))) {
+		$func = $a->module . '_content';
+		if ((! $a->error) && (function_exists($func))) {
 			$arr = array('content' => $a->page['content']);
 			call_hooks($a->module . '_mod_content', $arr);
 			$a->page['content'] = $arr['content'];
-			$func = $a->module . '_content';
 			$arr = array('content' => $func($a));
 			call_hooks($a->module . '_mod_aftercontent', $arr);
 			$a->page['content'] .= $arr['content'];
 		}
 
-		if(function_exists(str_replace('-','_',current_theme()) . '_content_loaded')) {
-			$func = str_replace('-','_',current_theme()) . '_content_loaded';
+		$func = str_replace('-', '_', current_theme()) . '_content_loaded';
+		if ($func) {
```

Comment:
This PR does not only include moving files to proper namespace, also other rewrites/coding-convention which makes it harder to review. :-(